drv/lpc55-swd/build.rs

```diff
@@ -29,22 +29,67 @@ fn generate_swd_functions(config: &TaskConfig) -> Result<()> {
 
     // The RoT -> SP SWD control requires setting the IO functions at runtime
     // as opposed to just startup.
+    //
+    // It turns out to be very expensive to call into the GPIO task for
+    // changing pin direction so these functions access the IOCON block
+    // directly.
     writeln!(
         &mut file,
         "{}",
         quote::quote! {
 
+        // This does the conversion of the pin configuration in a way
+        // that's compatible with the existing iocon APIs
+        fn to_conf(
```

Comment:
Is there a convenient way to share code with `lpc55-gpio-api`, which has the same logic in `iocon_configure`?